main.go

```diff
@@ -111,6 +111,9 @@ func main() {
 		os.Exit(0)
 	}
 
+	// function used to save panic to a file (WIP)
+	defer panicToFile()
+
```

Comment:
To try the feature I added
`panic("test")` after the `defer` and the file generated contains this stacktrace:
```
stacktrace from panic: 
goroutine 1 [running, locked to thread]:
runtime/debug.Stack(0x0, 0x0, 0x0)
	/home/linuxbrew/.linuxbrew/Cellar/go/1.15.3/libexec/src/runtime/debug/stack.go:24 +0xa5
main.panicToFile()
	/home/umberto/Nextcloud/8tb/Lavoro/arduino-create-agent/main.go:165 +0x22d
panic(0xa09400, 0xb0ebb0)
	/home/linuxbrew/.linuxbrew/Cellar/go/1.15.3/libexec/src/runtime/panic.go:975 +0x4c6
main.main()
	/home/umberto/Nextcloud/8tb/Lavoro/arduino-create-agent/main.go:117 +0x98
```

And this is stderr:
```
panic: test [recovered]
	panic: test

goroutine 1 [running, locked to thread]:
main.panicToFile()
	/home/umberto/Nextcloud/8tb/Lavoro/arduino-create-agent/main.go:170 +0x49b
panic(0xa09400, 0xb0ebb0)
	/home/linuxbrew/.linuxbrew/Cellar/go/1.15.3/libexec/src/runtime/panic.go:975 +0x4c6
main.main()
	/home/umberto/Nextcloud/8tb/Lavoro/arduino-create-agent/main.go:117 +0x98
```